pax/config/XENON1T.ini

```diff
@@ -262,18 +262,16 @@ pmt_afterpulse_types = {
     }
 
 # Example S2 afterpulse model from Qing
+# values extracted from Kevin's note:
+# xenon:xenon1t:micheneau:single_electron_study_xenon1t
+# although the exponential constant here 
+# is affected by electron lifetime.
+# But elife is not implemented in fax
+# we use a data-driven overall time profile
 s2_afterpulse_types = {
-    'Gate':  {'p': 4.3*(10**(-5)),
-            'time_distribution': 'normal',
-            'time_parameters': dict(loc=2.0 * us,
-                                    scale=0.2 * us)},
-    'Cathode':  {'p': 4.3*(10**(-5)),
-              'time_distribution': 'normal',
-              'time_parameters': dict(loc=182. * us,
-                                      scale=10 * ns)},
-    'Volume':  {'p': 4.3*(10**(-4)),
+    'Volume':  {'p': 5.4*(10**(-3)),
```

Comment:
Because Kevin had a cut on drift time when plotting Fig.4. So need to scale a little to accommodate the whole drift range after pulses.